code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -14,11 +14,12 @@ var/list/ai_list = list()
 
 /mob/living/silicon/ai
 	name = "AI"
-	icon = 'icons/mob/AI.dmi'//
+	icon = 'icons/mob/AI.dmi'
 	icon_state = "ai"
 	anchored = 1
 	density = 1
 	status_flags = CANSTUN|CANPUSH
+	a_intent = "harm" //so we always get pushed instead of trying to swap

```

Comment:
ALERT!!! There needs to be a way for the AI player to change this, because it does have an effect on AI Mech pilots!